pay-api/src/pay_api/services/payment_account.py

```diff
@@ -484,7 +484,10 @@ def _handle_payment_details(cls, account_request, is_sandbox, pay_system, paymen
         cfs_account: CfsAccountModel = CfsAccountModel.find_effective_by_account_id(payment_account.id) \
             if payment_account.id else None
         if pay_system.get_payment_system_code() == PaymentSystem.PAYBC.value:
-            if cfs_account is None:
+            if cfs_account is None or (payment_account.payment_method == PaymentMethod.EFT and cfs_account):
```

Comment:
when Switching from PAD to EFT, we always create a new cfs account, cause the previous EFT cfs account suppose to be inactivated when it switch to PAD, and also I can't just remove the payment details use cfs service, it returns 404.

Reply:
Wouldn't the above technically be something good for a comment?